open-company/open-company-web
# Dashboard layout

Move topics-columns component to Rum. Rename it to dashboard-layout to be consistent.

To test:
- With an org that has at least one board, visit /org
- [x] Do you see only the loading screen before the board appear? Good
- With an org w/o boards, load /org
- [x] Do you see the empty org illustration with the CTA to add a board? Good
- With an org that has at least 2 boards, load all posts directly (no internal nav)
- [x] Now click on a board, do you NOT see the empty board before the posts are loaded? Godd
- Go to all posts (directly)
- Click pencil edit on a post
- Edit and save
- [x] Do you still see All Posts highlighted in the left nav bar (behind the opac bg)? Good
- [x] Close the X to dismiss, are you again on All Posts? Good
- Go to a board
- [x] Repeat edit and save, do you see the board highlighted in left nav bar? Good
- [x] Dismiss the modal, are you still in the same board? Good
- Merge! 🚀
